### PR TITLE
Organize code

### PR DESCRIPTION
**Pull request by tymefighter**

### CODE REVIEW

File: kernel/buffer.c [hunk 1]
```diff
@@ -0,0 +1,70 @@
+#include "declarations.h"
+#include "functions.h"
+
+// Initialize Buffer Cache
```
Comment (sahul343) [inline kernel/buffer.c:4]: You can add a small description of what this file is responsible for

File: kernel/declarations.h [hunk 2]
```diff
@@ -0,0 +1,361 @@
...
+// System Parameters
+#define NREG        31                  // Number of Registers
+#define BSIZE       1024                // Disk Block Size
+#define PGSIZE      4096                // Bytes per Page
+#define PGSHIFT     12                  // Bits of Offset within a Page
+#define FSSIZE    1000                // size of disk in blocks
+#define CPUID       0
+#define MAXOPBLOCKS 10                  // max # of blocks any op writes
+
+// Machine Status Register, mstatus
```
Comment (sahul343) [inline kernel/declarations.h:30]: You can write one-two words for every machine status register

File: kernel/buffer.c (hunk 1 above)
Comment (tymefighter) [reply kernel/buffer.c:4]: I'll do it

File: kernel/declarations.h (hunk 2 above)
Comment (tymefighter) [reply kernel/declarations.h:30]: I'll do it

**Pull request merged**